ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+ShareActivity.swift

```diff
@@ -232,7 +232,8 @@ extension BrowserViewController {
                       .union(.newlines)
                       .union(.controlCharacters)
                       .union(.illegalCharacters)
-                    let filename = webView.title?.components(separatedBy: validFilenameSet).joined()
+                    let filename =
+                      webView.title?.components(separatedBy: validFilenameSet).joined() ?? ""
```

Comment:
Shouldn't this coalescing nil be `"Untitled"` not empty string (and removed from below usage of `filename`, now gives a warning below that right-side is never used)

Reply:
This is actually a bad cherry-pick I think 😂, in my other branch I have this accessing via `tab` and you can see there's nothing except formatting and adding `?? ""` to the end

Reply:
reverted it here https://github.com/brave/brave-core/pull/27953/commits/55e603b2d8b2004b83808ca94377005badce03b4